bitnami/argo-cd/templates/application-controller/service.yaml

```
@@ -35,12 +35,12 @@ spec:
   sessionAffinityConfig: {{- include "common.tplvalues.render" (dict "value" .Values.controller.service.sessionAffinityConfig "context" $) | nindent 4 }}
   {{- end }}
   ports:
-    - name: tcp-controller
-      port: {{ .Values.controller.service.port }}
+    - name: http-metrics
+      port: {{ coalesce .Values.controller.service.port .Values.controller.service.ports.metrics }}
       targetPort: controller
```

The targetPort was renamed to `metrics`
```suggestion
      targetPort: metrics
```